pkgs/top-level/python-packages.nix

```diff
@@ -8123,6 +8123,7 @@ self: super: with self; {
 
   pygame = callPackage ../development/python-modules/pygame {
     inherit (pkgs.darwin.apple_sdk.frameworks) AppKit;
+    SDL2 = pkgs.SDL2_2_24;
     SDL2_image = pkgs.SDL2_image_2_0_5;
```

Comment:
Same with SDL2_image here

Reply:
What do you mean?

Reply:
SDL2_image is used both by `toppler` and `pygame`. So you are suggesting to duplicate the library and hide it?